### ISSUE TITLE
Custom Dashboard

### ISSUE THREAD

**Issue by JennyPower** [elementor/elementor#15541]
It would be great to be able to override the WordPress dashboard with a template in much the same way you set up a maintenance page.

I know there are a few plugins that already do this, but it seems like only a small step to get Elementor to do it natively.

**Comment by DorShahar** [comment]
Thanks for the feedback,
What is the motivation? What do you wish to see in the dashboard? Is the content related only to Elementor? 

Thanks.

**Comment by sanjupanikkar** [comment]
Personally, I'd prefer such features remain in third-party plugins rather than add more bloat to Elementor.

Regardless, to answer @DorShahar the main motivation is to design and display a custom dashboard to clients. Already there are some plugins that allow inserting a single page Elementor template. Almost all the widgets work properly, which is great! However, the layout sometimes conflicts with WP admin CSS and the global site settings don't load so the widgets have to be styled manually.

**Comment by JennyPower** [comment]
@DorShahar Yes as @sanjupanikkar said: to provide a custom dashboard for client information. I know there are plugins available but it seems like it would only be a small step for Elementor to do natively as it's much the same concept as the maintenance page. Although I didn't realise the css conflicts that @sanjupanikkar pointed out.

**Comment by DorShahar** [comment]
Thanks for answering @JennyPower , @sanjupanikkar 
What kind of information on this dashboard do you wish to display?

**Comment by JennyPower** [comment]
@DorShahar Contact details for client to get support. Instructions on how to use the website. Internal links to adding content, maybe embedded instructional vids

**Comment by paauserrat** [comment]
I will need to display button to directly create a port, or upload media and those things too
